mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -32,37 +32,37 @@ namespace omp {
 
 struct AlignedClauseOps {
   llvm::SmallVector<Value> alignedVars;
-  llvm::SmallVector<Attribute> alignmentAttrs;
+  llvm::SmallVector<Attribute> alignments;
 };
 
 struct AllocateClauseOps {
-  llvm::SmallVector<Value> allocatorVars, allocateVars;
+  llvm::SmallVector<Value> allocateVars, allocatorVars;
 };
 
 struct CancelDirectiveNameClauseOps {
-  ClauseCancellationConstructTypeAttr cancelDirectiveNameAttr;
+  ClauseCancellationConstructTypeAttr cancelDirective;
 };
 
 struct CollapseClauseOps {
-  llvm::SmallVector<Value> loopLBVar, loopUBVar, loopStepVar;
+  llvm::SmallVector<Value> collapseLowerBound, collapseUpperBound, collapseStep;
 };
 
 struct CopyprivateClauseOps {
   llvm::SmallVector<Value> copyprivateVars;
-  llvm::SmallVector<Attribute> copyprivateFuncs;
+  llvm::SmallVector<Attribute> copyprivateSyms;
 };
 
 struct CriticalNameClauseOps {
-  StringAttr criticalNameAttr;
+  StringAttr symName;
```

Comment:
This matches the tablegen name of the corresponding argument. Originally, my intention was to rename both to `criticalSym`/`critical_sym`, but the problem is that the tablegen name must remain `sym_name` because it's a requirement of the `Symbol` trait of the `CriticalDeclareOp` that contains this clause.

I think we have to make an exception to this name, since it's forced by the trait. Maybe it'd make sense to move the trait to the clause so that it's self-contained. Let me know what you think.